packages/FormElement/stories/FormElement.stories.mdx

```diff
@@ -0,0 +1,199 @@
+
+The `<FormElement.Help />` will add an icon with a tooltip next to the label.
+
+In addition, a `<label />` is rendered by the `<FormElement />` and is associated to the input field by the `idForLabel` parameter of the same `children` function.
+
+```jsx
+<FormElement label="Form Label">
+  <FormElement.Help>
+    Give me some help. <a href="wegalvanize.com">Learn more</a>.
+  </FormElement.Help>
+  <FormElement.Instructions>
+    <span>
+      Example text for extra panel for questionnaires. Example text for extra panel for questionnaires Example text for
+      extra panel for questionnaires. Example text for extra panel for questionnaires
+    </span>
+  </FormElement.Instructions>
+  <FormElement.Content>
+    {({ idForLabel, ariaDescribedBy }) => (
+      <Input id={idForLabel} placeholder="Form placeholder" aria-describedby={ariaDescribedBy} />
+    )}
+  </FormElement.Content>
+  <FormElement.Description>
+    <span>This is description text</span>
+  </FormElement.Description>
+  <FormElement.Error>There was an error</FormElement.Error>
+</FormElement>
+```
+
+##### Result
+
+<FormElementStory>
+  <AccessibilityExample />
+</FormElementStory>
+
+### Basic Input Components
+
+The `<FormElement>` can be used with different components to fulfill a variety of input needs. This can be achieved by including basic input components such as the `<Select>`, `<Textarea>`, `<Checkbox>`, etc. as `children` for the `<FormElement.Content>`
+
+#### Textarea
+
+```jsx
+<FormElement hasFieldSet label="Form Label">
+  <FormElement.Content>
+    <Textarea defaultValue="Hello world">
+  </FormElement.Content>
+</FormElement>
+```
+
+##### Result
+
+<FormElementStory>
+  <FormElement hasFieldSet label="Form Label">
+    <FormElement.Help>
+      Give me some help. <a href="wegalvanize.com">Learn more</a>.
+    </FormElement.Help>
+    <FormElement.Content>
+      <Textarea defaultValue="Hello world" />
+    </FormElement.Content>
+  </FormElement>
+</FormElementStory>
+
+#### Checkbox
+
+```jsx
+const optionsArray = ["Black Panther", "Wonder Woman", "Spiderman", "The Incredibles", "Thor", <span>test</span>];
+
+const getCheckboxOptions = ariaDescribedBy =>
+  optionsArray.map(hero => (
+    <Checkbox
+      ariaDescribedBy={ariaDescribedBy}
+      key={hero}
+      onChange={() => {}}
+      isDisabled={false}
+      size={Checkbox.types.size.MEDIUM}
+    >
+      {hero}
+    </Checkbox>
+  ));
+
+<FormElement hasFieldSet label="Form Label">
+  <FormElement.Content>
+    {({ ariaDescribedBy: ariaDescribedByOuter }) => getCheckboxOptions(ariaDescribedByOuter)}
```

Comment:
🤔  While using a `map()` is probably the way consumers will really use this, I wonder if it's simpler to just dump in the actual children.  Like:

```jsx
<Checkbox ariaDescribedBy={ariaDescribedBy} onChange={() => {}}>Black Panther</Checkbox>
<Checkbox ariaDescribedBy={ariaDescribedBy} onChange={() => {}}>Spiderman</Checkbox>
<Checkbox ariaDescribedBy={ariaDescribedBy} onChange={() => {}}>Wonder Woman</Checkbox>
```

Then we can remove the array and function above and focus on the API.  
What do you think?